### PR TITLE
Readme, test tweaks, prepare to release

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: checked_yaml/example/example.dart [hunk 1]
```diff
@@ -36,10 +36,18 @@ class Configuration {
 }
 
 void main(List<String> arguments) {
-  final fileContents = File(arguments.single).readAsStringSync();
+  var sourceUrl = arguments.single;
```
Comment (jakemac53) [inline checked_yaml/example/example.dart]: nit: it looks like this is either a source url or a yaml string, consider renaming it if so? `sourceUrlOrYaml` or something.

Or, just drop support for one or the other, I think it distracts from the example.

File: checked_yaml/test/example_test.dart [hunk 2]
```diff
@@ -2,171 +2,111 @@
...
-import 'package:path/path.dart' as p;
+import 'package:checked_yaml/checked_yaml.dart';
 import 'package:test/test.dart';
-import 'package:test_descriptor/test_descriptor.dart' as d;
-import 'package:test_process/test_process.dart';
 
-void main() {
-  test('valid input', () async {
-    final proc = await _run('{"name": "bob", "count": 42}');
+import '../example/example.dart' as example;
```
Comment (jakemac53) [inline checked_yaml/test/example_test.dart]: This is pretty naughty, you won't ever be able to use `build_vm_compilers` if you do this, but :man_shrugging:

Comment (kevmoo) [reply checked_yaml/test/example_test.dart]: agreed 🤷‍♂

File: checked_yaml/example/example.dart (hunk 1 above)
Comment (kevmoo) [reply checked_yaml/example/example.dart]: It's kinda nice to support either – at least it makes my tests work.

Will rename.

Comment (kevmoo) [reply checked_yaml/example/example.dart]: fixed!

File: checked_yaml/test/example_test.dart (hunk 2 above)
Comment (kevmoo) [reply checked_yaml/test/example_test.dart]: @jakemac53 to file a bug!

**Pull request merged**